### DISCUSSION TITLE
Would you like to have the setting to switch off autoscrolling in the chat?

### DISCUSSION THREAD

**Discussion by ruff3d** [pieces-app/support#230] [Polls]
Whenever the GPT generates the answer (and usually it's long), the screen of the chat starts jumping one by line while it proceeds to enhance the answer, which irritates you and simply doesn't allow you to read until it ends.

**Comment by mack-at-pieces** [comment]
Hi @ruff3d - Totally here you on this! The team is working on some adjustments to the scroll-handling to hopefully make this a better experience. We appreciate you documenting this here and your patience as we work through these updates 👍🏼
